ledger/ledger-api-bench-tool/src/main/scala/com/daml/ledger/api/benchtool/Cli.scala

```diff
@@ -26,30 +30,49 @@ object Cli {
         },
       opt[Config.StreamConfig]("consume-stream")
         .abbr("s")
+        .unbounded()
+        .minOccurs(1)
         .text(
           s"Stream configuration."
         )
-        .valueName("streamType=<transactions|transaction-trees>,name=<streamName>,party=<party>")
-        .action { case (streamConfig, config) => config.copy(streamConfig = Some(streamConfig)) },
+        .valueName(
+          "stream-type=<transactions|transaction-trees>,name=<streamName>,party=<party>[,begin-offset=<offset>][,end-offset=<offset>][,template-ids=<id1>|<id2>]"
+        )
+        .action { case (streamConfig, config) =>
+          config.copy(streams = config.streams :+ streamConfig)
+        },
+      opt[FiniteDuration]("log-interval")
+        .abbr("r")
+        .text("Stream metrics log interval.")
+        .action { case (period, config) => config.copy(reportingPeriod = period) },
+      opt[Int]("core-pool-size")
+        .text("corePoolSize parameter of the ThreadPoolExecutor.")
+        .optional()
+        .action { case (size, config) =>
+          config.copy(concurrency = config.concurrency.copy(corePoolSize = size))
+        },
+      opt[Int]("max-pool-size")
+        .text("maxPoolSize parameter of the ThreadPoolExecutor.")
+        .optional()
+        .action { case (size, config) =>
+          config.copy(concurrency = config.concurrency.copy(maxPoolSize = size))
+        },
+      opt[Long]("keep-alive-time")
+        .text("keepAliveTime parameter of the ThreadPoolExecutor.")
```

Comment:
Do we really need to expose such a parameter? It is unlikely anyone will ever need to change it. It makes CLI less transparent

Reply:
You're right. I'll remove this one.